<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.2">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2.63">
    <parentIds>
      <string>65</string>
    </parentIds>
    <id>66</id>
    <startId>10</startId>
  </node>
  <actions>
    <wf.a.TimingAction plugin="workflow-api@2.33">
      <startTime>1550226969827</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@2.33">
      <error class="hudson.AbortException">
        <detailMessage>script returned exit code 1</detailMessage>
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.handleExit(DurableTaskStep.java:561)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.check(DurableTaskStep.java:507)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.durable_task.DurableTaskStep$Execution.run(DurableTaskStep.java:453)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)</trace>
          <trace>java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
          <c class="list"/>
          <list reference="../c"/>
        </suppressedExceptions>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>